docs/guides/Getting-Started/getting-started/marketplace-overview.md

```diff
@@ -0,0 +1,177 @@
+
+After configuring the account, the marketplace will be able to verify the settings made.
+
+- GET - [List Sellers](https://developers.vtex.com/docs/api-reference/marketplace-apis#get-/seller-register/pvt/sellers): Retrieve a list of sellers registered in the marketplace, and this list can be filtered by the configured [trade policy](https://help.vtex.com/en/tutorial/como-funciona-uma-politica-comercial--6Xef8PZiFm40kg2STrMkMV#master-data).
+- GET - [Get Seller data by ID](https://developers.vtex.com/docs/api-reference/marketplace-apis#get-/seller-register/pvt/sellers/-sellerId-): Retrieve information on a specific seller, it is also possible to filter by commercial policy.
+- GET - [Get Sales Channel Mapping Data](https://developers.vtex.com/docs/api-reference/marketplace-apis/#get-/seller-register/pvt/sellers/-sellerId-/sales-channel/mapping): Retrieve information about mapping between your sales channels and a specific seller.
+
+### Configuring seller's commission
+
+The marketplace will configure the seller's commissions after the initial configurations. To perform this action, one of the following endpoints can be used.
+
+- PUT - [Upsert Seller Commissions in Bulk](https://developers.vtex.com/docs/api-reference/marketplace-apis#put-/seller-register/pvt/sellers/-sellerId-/commissions/categories): To apply the same commission rule to all categories of a seller already registered in the marketplace.
+- PUT - [Upsert Seller Commissions by Category ID](https://developers.vtex.com/docs/api-reference/marketplace-apis#put-/seller-register/pvt/sellers/-sellerId-/commissions/-categoryId-): To apply a commission rule to only one category of the respective seller.
+
+The query and management of applied commission rules will be performed through the following endpoints.
+
+- GET - [List Seller Commissions by seller ID](https://developers.vtex.com/docs/api-reference/marketplace-apis/#get-/seller-register/pvt/sellers/-sellerId-/commissions): Retrieve the list of commissions defined for a seller.
+- GET- [Get Seller Commissions by Category ID](https://developers.vtex.com/docs/api-reference/marketplace-apis/#get-/seller-register/pvt/sellers/-sellerId-/commissions/-categoryId-): Retrieve the commission list defined for a specific category of a specific seller.
+- DELETE - [Remove Seller Commission by Category ID](https://developers.vtex.com/docs/api-reference/marketplace-apis#delete-/seller-register/pvt/sellers/-sellerId-/commissions/-categoryId-): Removes a previously defined commission for a category of a seller.
+
+When completing the steps listed above, the marketplace will have completed the integration of a seller into its store.
+
+## 2. Suggestions
+
+When the seller is integrated into a store, he will begin to suggest products from his catalog to the marketplace. The APIs mentioned in this section refer to the Received SKUs page of Admin VTEX.
```

Comment:
```suggestion
When the seller is integrated into a marketplace, it will begin to suggest products from its catalog to the marketplace. The APIs mentioned in this section refer to the [Received SKUs](https://help.vtex.com/en/tutorial/manual-sku-cataloging--tutorials_396) page of VTEX Admin.
```